lsp-mode.el

```diff
@@ -928,7 +928,7 @@ must be used for handling a particular message.")
 (defconst lsp--file-change-type
   `((created . 1)
     (changed . 2)
-    (deleted . 3)))
+    (deleted . 4)))
```

Comment:
This should be 3 according to https://microsoft.github.io/language-server-protocol/specification#workspace_didChangeWatchedFiles, so I don't think you can change it to 4 to allow bitwise operations.

Reply:
Oh, I didn't see that part of the spec. I mistook `WatchFile` events (which proceed in 1, 2, 4 order) for these events.